akka-docs/src/main/paradox/typed/reliable-delivery.md

```diff
@@ -65,33 +65,34 @@ The `ProducerController` sends `RequestNext` to the producer, which is then allo
 message to the `ProducerController`. Thereafter the producer will receive a new `RequestNext`
 when it's allowed to send one more message.
 
-The producer and `ProducerController` actors are supposed to be local so that these messages are
-fast and not lost. This is enforced by a runtime check.
+The producer and `ProducerController` actors are required to be local so that message delivery is
+both fast and guaranteed. This requirement is enforced by a runtime check.
 
 Similarly, on the consumer side the destination consumer actor will start the flow by sending an
 initial `ConsumerController.Start` message to the `ConsumerController`. 
 
-For the `ProducerController` to know where to send the messages it must be connected with the
-`ConsumerController`. You do this is with `ProducerController.RegisterConsumer` or
-`ConsumerController.RegisterToProducerController` messages. When using the the point-to-point pattern
-it is the application's responsibility to connect them together. For example, by sending the `ActorRef`
-in an ordinary message to the other side, or register the `ActorRef` in the @ref:[Receptionist](actor-discovery.md)
-and find it on the other side.
+For the `ProducerController` to know where to send the messages, it must be connected with the
+`ConsumerController`. This can be done with the `ProducerController.RegisterConsumer` or
+`ConsumerController.RegisterToProducerController` messages. When using the point-to-point pattern,
+it is the application's responsibility to connect them together. For example, this can be done by sending the `ActorRef`
+in an ordinary message to the other side, or by registering the `ActorRef` in the @ref:[Receptionist](actor-discovery.md)
+so that it can be found on the other side.
 
 You must also take measures to reconnect them if any of the sides crashes, for example by watching it
 for termination.
 
-Received messages from the producer are wrapped in `ConsumerController.Delivery` when sent to the consumer,
-which is supposed to reply with `ConsumerController.Confirmed` when it has processed the message.
-Next message is not delivered until the previous is confirmed. More messages from the producer that arrive
+Messages sent by the producer are wrapped in `ConsumerController.Delivery` when received  by a consumer and the consumer
```

Comment:
extra space between received and by

Reply:
Fixed ([d7b83b9](https://github.com/akka/akka/pull/28738/commits/d7b83b94b8efb467b0bf248a96be580f0b716db6))